app/routes/_gcn.user.tsx

```diff
@@ -40,6 +40,9 @@ export default function () {
                     </NavLink>,
                   ]
                 : []),
+              <NavLink key="circulars" to="circulars">
+                Circulars
+              </NavLink>,
```

Comment:
Why have we added this to the user menu?

Reply:
Oops, leftover from an idea i was trying. Will remove